components/image_cropping/src/main.py

```diff
@@ -17,7 +18,7 @@
 logger = logging.getLogger(__name__)
 
 
-def extract_dimensions(image_df: dd.DataFrame) -> [np.int16, np.int16]:
+def extract_dimensions(image_df: dd.DataFrame) -> t.Union[np.int16, np.int16]:
```

Comment:
Doesn't `union` suggest that it's either one or the other?

Suggestion: 
```
def extract_dimensions(image_df: dd.DataFrame) -> t.Tuple[np.int16, np.int16]:
    """Extract the width and height of an image

    Args:
        image_df (dd.DataFrame): input dataframe with images_data column

    Returns:
       Tuple containing the width and height of the images
    """
    image = Image.open(io.BytesIO(image_df["images_data"]))

    return np.int16(image.size[0]), np.int16(image.size[1])
```

Reply:
Yes, wasn't paying attention apparently :sweat_smile:

Reply:
You were doing a lot of heavy lifting yesterday so you're good :P